homeassistant/components/vicare/climate.py

```diff
@@ -49,6 +49,10 @@
 SERVICE_SET_VICARE_MODE = "set_vicare_mode"
 SERVICE_SET_VICARE_MODE_ATTR_MODE = "vicare_mode"
```

Comment:
I wonder if this should not have been a `Select` entity instead.

Reply:
see other reponse